src/logic/tokens/utils/__tests__/formatAmount.test.ts

```diff
@@ -107,71 +107,80 @@ describe('formatAmount', () => {
   })
 })
 
-describe('FormatsAmountsInUsFormat', () => {
+describe('formatCurrency', () => {
   it('Given 0 returns 0.00', () => {
     // given
     const input = 0
-    const expectedResult = '0.00'
+    const expectedResult = '0.00 EUR'
 
     // when
-    const result = formatAmountInUsFormat(input.toString())
+    const result = formatCurrency(input.toString(), 'EUR')
```

Comment:
Curious if this tests will pass if running with a different locale. I runned them locally tweaking the `LOCALE` between `'de'` and `'en'` in https://github.com/gnosis/safe-react/blob/6e99be5694abcd4587333e0f855e069a4d1710aa/src/logic/tokens/utils/formatAmount.ts#L6  Apparently they do 🤯

Reply:
Node only ships with a handful of locales and defaults to those. Hence, the `'de'` format defaulting to `'en'` in the test environment. In the browser, however, you should see it formatted as `'de-DE'`.

In order to support all locales in the test environment, we'd need to install [a another package](https://www.npmjs.com/package/full-icu) just for these two functions. In my opinion, that is also focused on testing `Intl.NumberFormat`, not the functions in question.

The tests in question are legacy and are passing (albeit defaulting to English). I feel as though sufficiently cover the functionality as it's possible to fully confirm the non-English locales in the browser.

What do you think?

Reply:
I agree to leave the tests as they are and cover this matter potentially in a E2E test (out of scope).

I also think leaving a comment in the tests would avoid this question in the future. 

Thank you for the further research @iamacook